microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -314,16 +316,17 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
 <!-- ====================================================================== -->
 ## Import browser data
 
-Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time.  You can import your browser data to Microsoft Edge from another browser.  During import, you can either keep your existing data, or delete it and start fresh.  The data includes your favorites, browsing history, autofill data, extensions, settings, and other browsing data.
+Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time. You can import your browser data to Microsoft Edge from another browser. The data includes your favorites, browsing history, cookies, autofill data, extensions, settings, and other browsing data.
 
-Your browsing data from older versions of Microsoft Edge is automatically imported when you update Microsoft Edge.  With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  Importing your data is completed locally on your device, and is stored locally, and is not sent to Microsoft unless you sign in and sync your browsing data.
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system. Importing your data is completed locally on your device, and is stored locally, and is not sent to Microsoft unless you sign in and sync your browsing data.
 
-![Import.](./media/migration.png)
+You can manage your import preferences any time from `edge://settings/profiles/importBrowsingData`
 
-You can import extensions from a different browser such as Google Chrome. If the extension is not available on the Microsoft Edge Add-ons website, Microsoft Edge imports a local copy and asks for permission before starting. The permissions for some of the extensions may have changed. To review the extension permissions, go to `edge://extensions`.
+![Import.](./media/migration.png)
```

Comment:
is the name "Trevor Philips" approved?  Replace it by white block to erase it?

About the list of approved names: check:

internal version of Docs Contributor Guide
corp style guide

Smaller public versions of those:
https://learn.microsoft.com/en-us/contribute/
https://learn.microsoft.com/en-us/style-guide/welcome/

Reply:
@captainbrosset - pls check whether "Trevor Philips" is in the Approved Names database.  If not, please blank it out, or do another fix:
https://github.com/MicrosoftDocs/edge-developer/raw/user/kbilardello/privacy-whitepaper-updates/microsoft-edge/privacy-whitepaper/media/migration.png

Reply:
instead of manually using the above png, I created PR into the present PR.  So this thread can be Resolved.